modules/exploits/multi/http/spring_framework_rce_spring4shell.rb

```diff
@@ -172,27 +166,33 @@ def fix(jsp)
   end
 
   def create_jsp
+    jsp = <<~EOS
+      <%
+        File jsp=new File(getServletContext().getRealPath(File.separator) + File.separator + "#{@jsp_file}");
+        jsp.delete();
+      %>
+      #{Faker::Internet.uuid}
```

Comment:
This means there's a non-empty response when the payload has been run which is used by the retry loop to abort the wait.